src/extra/libs/gif/lv_gif.h

```diff
@@ -35,6 +35,10 @@ typedef struct {
     uint32_t last_call;
 } lv_gif_t;
 
+typedef struct {
```

Comment:
Seemingly it's not used anywhere. 

(I couldn't use the frame_id feature of the decoders as I couldn't decode an arbitrary frame with the gif decoder, only the next one.

Reply:
Removed.